assignments/objects.js

```diff
@@ -18,28 +18,76 @@ const example = {
 }
 
 // Write your intern objects here:
-
-
+const mitzi = {
+id:0,
+name: 'Mitzi,F',
+email: "mmelloy0@psu.edu",
+Gender:"F"
+};
+
+console.log(mitzi);
+
+const kennan= {
+  id:1,
+  name: 'Kennan,M',
+  email: "kdiben1@tinypic.com",
+  Gender:"M"
+  
+}
+  console.log(kennan);
+
+const keven= {
+  id:2,
+  name: 'Keven,M',
+  email: "kmummery2@wikimedia.org",
+  Gender:"M"
+  };
+  console.log(keven);
+  
+  const gannie = {
+    id:3,
+    name: 'Gannie,M',
+    email: "gmartinson3@illinois.edu",
+    Gender:"F"
+    };
+console.log(gannie);
+
+    const antonietta= {
+      id:4,
+      name: 'Antonietta,F',
+      email: "adaine5@samsung.com",
+      Gender:"F"
+      };
+      console.log(antonietta);
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+mitzi.name;
 // Kennan's ID
-
+kennan.id;
 // Keven's email
-
+keven.email;
 // Gannie's name
-
+gannie.name;
 // Antonietta's Gender
+antonietta.Gender;
```

Comment:
```suggestion
antonietta.gender;
```
Try to make object keys have lower case names to stick to naming conventions :+1:

Reply:
:heavy_check_mark: